NMEA2000.h

```diff
@@ -345,6 +349,13 @@ inline void SetN2kProductInformation(tN2kMsg &N2kMsg, unsigned int N2kVersion, u
                   SertificationLevel,LoadEquivalency);
 }
 
+
+void SetN2kPGN59904(tN2kMsg &N2kMsg, uint8_t Destination, unsigned long RequestedPGN);
+
+inline void SetN2kPGNISORequest(tN2kMsg &N2kMsg, uint8_t Destination, unsigned long RequestedPGN) {
```

Comment:
Why we would't define
    // ISO Request handler. Handler must return true, if ith has handled request completely
    // Handler should respond to Destination
    bool(*ISORequestHandler)(unsigned long RequestedPGN, unsigned char Destination, int DeviceIndex);

   Then handler does not need to parse request msg again.
   DeviceIndex is the id for multidevice system, which is still under construction.